wordpress-mobile/WordPress-iOS
# Testing: Add email signup UI test

This PR adds an email signup test (the final UI test in the initial set outlined in https://github.com/wordpress-mobile/WordPress-iOS/issues/11013).

It relies on the following open PRs:

* ~https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/102~
* ~https://github.com/wordpress-mobile/WordPressMocks/pull/7~

To test:

* Run `rake dependencies` to confirm you have the required pod updates (noted above).
* Run `rake mocks` to start the mock server.
* Open the simulator.
* Run the tests in the `WordPressUITests` target. Confirm the signup test goes through the complete email signup flow and passes on both iPhone and iPad. (You can also run the signup test on a simulator set to a non-English language to confirm it passes; note that the editor tests will not yet pass in other languages.)

Update release notes:
